tools/sklearn/main_macros.xml

```diff
@@ -3,8 +3,7 @@
 
   <xml name="python_requirements">
       <requirements>
-          <requirement type="package" version="3.6">python</requirement>
-          <requirement type="package" version="0.8.1">Galaxy-ML</requirement>
+          <requirement type="package" version="0.8.3">Galaxy-ML</requirement>
```

Comment:
@bgruening I noticed that the requirement for python3.6 is removed. I put it back (locally) and ran the tests again, and I got the "UnboundLocalError: local variable 'lib' referenced before assignment" error again. I am assuming (please confirm) that the python3.6 requirement is needed (and I deleted it by mistake). Anyhow, the test errors change depending on having or not having this requirement.

Reply:
It should not be needed as Galaxy-ML depends on python, so Galaxy-ML will pull in python.

Reply:
Ok, thats good.